value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -397,6 +387,26 @@ private void processType(TypeElement type) {
       }
     }
 
+    if (appliedExtensions.size() > 0) {
+      final Set<String> methodsToRemove = Sets.newHashSet();
+      for (int i = appliedExtensions.size() - 1; i >= 0; i--) {
+        AutoValueExtension extension = appliedExtensions.get(i);
+        Collection<String> consumed = extension.consumeProperties(context);
+        if (consumed != null) {
```

Comment:
Also, the test intentionally returns null since it's possible based on the interface and I need to test that case.